.evergreen/.evg.yml

```diff
@@ -861,11 +861,35 @@ tasks:
           vars:
             file: ".evergreen/static-checks.sh"
 
-    - name: "test"
+    - name: "test-bson-and-crypt"
+      commands:
+        - func: "run tests"
+          vars:
+            TESTS: 'bson:test bson-kotlin:test bson-kotlinx:test bson-scala:test bson-record-codec:test mongodb-crypt:test'
+
+    - name: "test-core-and-legacy"
+      commands:
+        - func: "start-kms-kmip-server"
+        - func: "bootstrap mongo-orchestration"
+        - func: "run tests"
+          vars:
+            TESTS: 'driver-core:test driver-legacy:test'
+
+    - name: "test-sync"
       commands:
         - func: "start-kms-kmip-server"
         - func: "bootstrap mongo-orchestration"
         - func: "run tests"
+          vars:
+            TESTS: 'driver-sync:test driver-kotlin-sync:test'
+
+    - name: "test-reactive"
+      commands:
+        - func: "start-kms-kmip-server"
+        - func: "bootstrap mongo-orchestration"
+        - func: "run tests"
+          vars:
+            TESTS: 'driver-reactive-streams:test driver-kotlin-coroutine:test'
```

Comment:
Collectively these take around 10 minutes to run on a replica set

Reply:
It's also advantageous to split out since reactive tests tend to be flakier than sync tests.